tests/expressions/tests.py

```diff
@@ -149,6 +149,80 @@ def test_update(self):
             ],
         )
 
+    def test_slicing_in_f_expressions(self):
+        # F expressions using string-fields can be sliced.
+        tests = [
+            (F('name')[:], 'Example Inc.', 'Example Inc.'),
+            (F('name')[:7], 'Example Inc.', 'Example'),
+            (F('name')[0], 'Example', 'E'),
+            (F('name')[5], 'E', ''),
+            (F('name')[7:], 'Foobar Ltd.', 'Ltd.'),
+            (F('name')[0:10], 'Ltd.', 'Ltd.'),
+            (F('name')[2:7], 'Test GmbH', 'st Gm'),
+        ]
+        for expression, name, expected in tests:
+            with self.subTest(expression=expression, name=name, expected=expected):
+                company = Company.objects.get(name=name)
+                company.name = expression
+                company.save()
+                company.refresh_from_db()
+                self.assertEqual(company.name, expected)
+
+    def test_slicing_in_f_expressions_with_annotate(self):
+        companies = Company.objects.annotate(first_three=F('name')[:3])
+        self.assertSequenceEqual(
+            companies.values_list('first_three', flat=True),
+            ['Exa', 'Foo', 'Tes']
```

Comment:
Please add a trailing comma in all cases:
```suggestion
            ['Exa', 'Foo', 'Tes'],
```